airflow/ui/src/layouts/Nav/UserSettingsButton.tsx

```diff
@@ -60,6 +63,9 @@ export const UserSettingsButton = () => {
           <FiClock size="1.25rem" style={{ marginRight: "8px" }} />
           {dayjs().tz(selectedTimezone).format("HH:mm z (Z)")}
         </Menu.Item>
+        <Menu.Item value="version">
+          Version <a href={`https://pypi.org/project/apache-airflow/${data?.version}/`} rel="noreferrer" style={{color: "#365f84"}} target="_blank">{data?.version}</a>
```

Comment:
We should probably link to `https://airflow.apache.org/docs/apache-airflow/{version}/` instead. We may want to only have it be a link if it's a normal release - not a dev/alpha/rc/etc.

Reply:
I see that there is a link for a stable version `https://airflow.apache.org/docs/apache-airflow/stable/index.html`. Should I link to this one instead?

Reply:
No. Stable always points the latest normal release, and it will be wrong most of the time unless the user always upgrade at the first oppertunity.

Reply:
got it!